### PR TITLE
Quick grad rename

### PR DESCRIPTION
**Pull request by jxmorris12**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -11,10 +11,21 @@ TextAttack
 Features 
 -----------
 
-- **Search Methods**: Explores the transformation space and attempts to find a successful attack
-- **Transformations**: Takes a text input and transforms it by replacing words and phrases while attempting to retain the meaning
-- **Constraints**: Determines if a given transformation is valid
-- **Built-in Datasets** and **Pre-trained Models** for ease of use
+TextAttack isn't just a Python library; it's a framework for constructing adversarial attacks in NLP. TextAttack builds attacks from four components:
+
+- **Goal Functions** stipulate the goal of the attack, like to change the prediction score of a classification model, or to change all of the words in a translation output
```
Comment (uvafan) [inline docs/index.rst:16]: It would be cool if Goal Functions linked to the Goal Function documentation, etc.

File: docs/index.rst [hunk 2]
```diff
@@ -11,10 +11,21 @@ TextAttack
...
+- **Search Methods** explores the space of transformations and attempt to find a successful perturbtion
+- **Transformations** takes a text input and transform it by replacing characters, words, or phrases
+- **Constraints**: Determines if a potential perturbtion is valid with respect to the original input
+
+TextAttack provides a set of **attack recipes** that assemble attacks from the literature from these four components.
+
+TextAttack has some other features that make it a pleasure to use:
```
Comment (uvafan) [inline docs/index.rst:23]: Mention data augmentation?

File: README.md [hunk 3]
```diff
@@ -39,7 +39,7 @@ environment variable `TA_CACHE_DIR`.
 
 ### Running Attacks
 
-The [`examples/`](examples/) folder contains notebooks walking through examples of basic usage of TextAttack, including building a custom transformation and a custom constraint.
+The [`examples/`](docs/examples/) folder contains notebooks walking through examples of basic usage of TextAttack, including building a custom transformation and a custom constraint. These examples can also be viewed through the documentation website.
```
Comment (uvafan) [inline README.md:42]: Include link to documentation page?

File: docs/index.rst (hunk 2 above)
Comment (jxmorris12) [reply docs/index.rst:23]: How could I forget!

File: docs/index.rst (hunk 1 above)
Comment (jxmorris12) [reply docs/index.rst:16]: agreed, we'll add this when we add the last stuff

**Pull request merged**